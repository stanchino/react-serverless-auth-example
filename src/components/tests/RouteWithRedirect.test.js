import React from "react";
import { RouteWithRedirect } from "..";
import { testConnectedContainer } from "./shared-examples";

describe("RouteWithRedirect", () => testConnectedContainer(<RouteWithRedirect path={'/auth/login'} to={'/'} />));