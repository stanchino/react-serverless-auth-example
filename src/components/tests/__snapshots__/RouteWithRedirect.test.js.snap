// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteWithRedirect renders without errors 1`] = `null`;
